crates/cxx-qt-lib/src/gui/qimage.rs

```diff
@@ -147,6 +147,11 @@ mod ffi {
         /// Returns the enclosing rectangle (0, 0, width(), height()) of the image.
         fn rect(self: &QImage) -> QRect;
 
+        /// Swaps the values of the red and blue components of all pixels, effectively converting an RGB image to an BGR image.
+        #[cfg(cxxqt_qt_version_major = "6")]
```

Comment:
guess this sortof thing will be better once we have the other change for `at_least_qt_version_6`

Reply:
I will merge with other patch it's better :)